packages/global/start-server.js

```diff
@@ -74,6 +74,13 @@ module.exports = (options = {}) => {
       // i18n
       i18n(app, options.i18n);
 
+      // Add IdentityX logoutHook to remove omeda_promo_code cookie
+      const identityX = getAsObject(options, 'siteConfig.identityX');
+      identityX.addHook({
+        name: 'onLogout',
+        fn: ({ res }) => res.clearCookie('omeda_promo_code'),
```

Comment:
This should use the Omeda+IdX config param rather than a hardcoded value. https://github.com/parameter1/base-cms/blob/master/packages/marko-web-omeda-identity-x/index.js#L32